docs/gallery/gallery/customization/legend.jl

```diff
@@ -18,6 +18,12 @@ draw(plt)
 
 fg = draw(plt, legend=(position=:top, titleposition=:left, framevisible=true, padding=5))
 
+# To adjust the title and order of labels in a legend you can use the pair syntax.
+
+layers = linear() +  mapping(color= :group => sorter(["a looooooong label", "and one more long label",  "an even loooooonger label"]) => "Labels")
```

Comment:
It may be cleaner to define a list of options at the beginning:

`labels = ["a looooooong label", "and one more long label",  "an even loooooonger label"]`

Then use `group=rand(labels, 100)` above and `mapping(color=:group => sorter(labels) => "Labels")` here.